xmrstak/cli/cli-miner.cpp

```diff
@@ -90,6 +89,170 @@ void help()
 	cout<<"Brought to by fireice_uk and psychocrypt under GPLv3."<<endl;
 }
 
+bool read_yes_no(const char* str)
+{
+	std::string tmp;
+	do
+	{
+		std::cout << str << std::endl;
+		std::cin >> tmp;
+		std::transform(tmp.begin(), tmp.end(), tmp.begin(), ::tolower);
+	}
+	while(tmp != "y" && tmp != "n" && tmp != "yes" && tmp != "no");
+
+	return tmp == "y" || tmp == "yes";
+}
+
+inline const char* bool_to_str(bool v)
```

Comment:
could you please use `std::string` as return type

Reply:
Same as above, this std::string here is just useless busywork.